src/main/java/org/sciborgs1155/robot/intake/Intake.java

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package org.sciborgs1155.robot.intake;
+
+import static org.sciborgs1155.robot.Ports.Intake.*;
+import static org.sciborgs1155.robot.intake.IntakeConstants.*;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMax.IdleMode;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.trajectory.TrapezoidProfile.State;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj2.command.TrapezoidProfileCommand;
+import org.sciborgs1155.lib.constants.SparkUtils;
+
+public class Intake extends SubsystemBase {
+
+  private final CANSparkMax rotateMotor =
```

Comment:
You should use the IO format for each component here, so something like:
- JointIO
  - RealJoint
  - SimJoint
  - NoJoint (preparing for the worst is never a bad thing)
- IntakeIO
  - RealIntake
  - SimIntake
  - NoIntake

Reply:
I can't justify why simulating the intake wheels would be important with its current functionality. Instead, in sim, a NoIntake() is created.